<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>300</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">#widget
{
	
border: 3px solid gray;
border-radius: 40px;
background: white;
}</string>
    </property>
    <widget class="QScrollArea" name="scrollArea">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>221</width>
       <height>221</height>
      </rect>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>198</width>
        <height>488</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="readImage">
         <property name="text">
          <string>读取图像</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="writeImage">
         <property name="text">
          <string>保存图像</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="apply">
         <property name="text">
          <string>应用</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="showOrigin">
         <property name="text">
          <string>显示原图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ImageInfo">
         <property name="text">
          <string>显示图像信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Histogram">
         <property name="text">
          <string>计算直方图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="binarization">
         <property name="text">
          <string>二值化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="normalization">
         <property name="text">
          <string>归一化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Inversion">
         <property name="text">
          <string>图像反色</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="graying">
         <property name="text">
          <string>转为灰度图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="linerTranslate">
         <property name="text">
          <string>线性变换</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="grayChange">
         <property name="text">
          <string>灰度化</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="grayStretch">
         <property name="text">
          <string>灰度拉伸</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pseudoColor">
         <property name="text">
          <string>真彩色转伪彩色</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>521</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>220</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="Width">
       <property name="text">
        <string>图像宽度：0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Height">
       <property name="text">
        <string>图像高度：0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="Fliter">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>滤镜</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>160</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>旋转</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDial" name="rotate">
       <property name="minimum">
        <number>-180</number>
       </property>
       <property name="maximum">
        <number>180</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>120</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>亮度</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="brightnessChange">
       <property name="minimum">
        <number>-255</number>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>460</y>
      <width>261</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>x轴</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="xAxis">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>y轴</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="yAxis">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>x轴缩放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Xscale">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>y轴缩放</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="Yscale">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>5</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="morphology">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>30</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>形态学计数</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>874</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
